ext/curl/lib/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_compile_definitions(HAVE_CONFIG_H CURL_NO_OLDIES)
+
+zephyr_library_sources(
+	altsvc.c amigaos.c asyn-ares.c asyn-thread.c base64.c
```

Comment:
I think a list like this will make it easier to see when new files are added / removed in commits:
```
  altsvc.c
  amigaos.c
  asyn-ares.c
  asyn-thread.c
  base64.c
  ....
```
but I acknowledge it also results in a very long list of files.

But as minimum, please sort the files alphabetic as it is hard to spot if a specific file is in the list or not.
Currently I see this:
```
... ftp.c url.c ftplistparser.c ...
... psl.c doh.c rand.c ...
```
and probably more.

Reply:
The list is copied from the Makefile.inc of the source repository. Having the list with same order and line breaks will help if we merge changes from the source repository. Personally, I'd like to keep the synchronization to the source repository file. What do you think?

Reply:
Fair point.

I agree that if this code is copied mostly as-is from external part then it's be to keep it that way.

A comment in the CMakeLists.txt about this would be helpful.

Reply:
Comment added.